upgrade/upgrade-scripts/common.sh

```diff
@@ -15,6 +15,13 @@
 # limitations under the License.
 #
 
+#
+# set umask to 0022 so that all new directories will be created with correct
+# permissions. This is especially required for not-in-place upgrades. This will
+# override the umask set by mgmt stack.
+#
+umask 0022
+
 # shellcheck disable=SC2034
```

Comment:
You should put this after the shellcheck instruction, otherwise the shellcheck instruction doesn't apply to the whole file (and you get a shellcheck failure)

Reply:
Got it, thanks